include/wx/generic/dirctrlg.h

```diff
@@ -48,10 +50,28 @@ enum
     wxDIRCTRL_EDIT_LABELS    = 0x0100,
     // Allow multiple selection
     wxDIRCTRL_MULTIPLE       = 0x0200,
+    // Enable right-click menu
+    wxDIRCTRL_RCLICK_MENU               = 0x0400,
```

Comment:
I don't really understand what purpose does this flag have, in isolation. It doesn't seem to do anything on its own, so why have it at all? Alternatively, and probably better, let's have this flag, but configure the menu contents in some other way (if necessary, I'm not even sure we need such flexibility). I.e. specifying this flag would should context menu with default contents in the control.